python/ray/util/client/server/server.py

```diff
@@ -307,45 +311,53 @@ def send_get_response(result: Any) -> None:
                         serialized = dumps_from_server(result, client_id, self)
                         get_resp = ray_client_pb2.GetResponse(
                             valid=True, data=serialized)
-                    except Exception as e:
+                    except Exception as exc:
                         get_resp = ray_client_pb2.GetResponse(
-                            valid=False, error=cloudpickle.dumps(e))
+                            valid=False, error=cloudpickle.dumps(exc))
                     resp = ray_client_pb2.DataResponse(
                         get=get_resp, req_id=req_id)
-                    resp.req_id = req_id
```

Comment:
isn't this still needed?

Reply:
It is already set with `req_id=req_id` above?

Reply:
Oops! Totally missed that :)